packages/manager/src/features/Linodes/LinodesDetail/LinodeConfigs/UpgradeInterfaces/DialogContents/ConfigSelectDialogContent.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button, Notice, Select, Stack } from '@linode/ui';
+import React from 'react';
+
+import {
+  CONFIG_SELECT_ACTUAL_UPGRADE_COPY,
+  CONFIG_SELECT_DRY_RUN_COPY,
+  UPGRADE_INTERFACES_WARNING,
+} from '../constants';
+import { useUpgradeToLinodeInterfaces } from '../useUpgradeToLinodeInterfaces';
+
+import type {
+  ConfigSelectDialogState,
+  UpgradeInterfacesDialogContentProps,
+} from '../types';
+
+export const ConfigSelectDialogContent = (
+  props: UpgradeInterfacesDialogContentProps<ConfigSelectDialogState>
+) => {
+  const { linodeId, onClose, setDialogState, state } = props;
+
+  const [selectedConfigId, setSelectedConfigId] = React.useState<
+    number | undefined
+  >();
+
+  const { upgradeToLinodeInterfaces } = useUpgradeToLinodeInterfaces({
+    linodeId,
+    selectedConfig: state.configs.find(
+      (config) => config.id === selectedConfigId
+    ),
+    setDialogState,
+  });
+
+  const configOptions = state.configs.map((config) => {
+    return { label: config.label, value: config.id };
+  });
+
+  return (
+    <Stack>
+      {state.isDryRun
+        ? CONFIG_SELECT_DRY_RUN_COPY
+        : CONFIG_SELECT_ACTUAL_UPGRADE_COPY}
+      {!state.isDryRun && selectedConfigId && (
+        <Notice
+          data-testid="upgrade-interfaces-warning"
+          sx={{ marginTop: 2 }}
+          variant="warning"
+        >
+          {UPGRADE_INTERFACES_WARNING}
+        </Notice>
+      )}
+      <Select
+        value={
+          configOptions.find((options) => options.value === selectedConfigId) ??
+          null
+        }
+        label="Configuration Profile"
+        onChange={(_, item) => setSelectedConfigId(item.value)}
+        options={configOptions}
+        placeholder="Select Configuration Profile"
+      />
+      <Stack direction="row-reverse" gap={2} sx={{ marginTop: 4 }}>
+        <Button
+          buttonType="primary"
+          disabled={!selectedConfigId}
+          onClick={() => upgradeToLinodeInterfaces(state.isDryRun)}
+        >
+          {state.isDryRun ? 'Upgrade Dry Run' : 'Upgrade Interfaces'}
+        </Button>
+        <Button buttonType="secondary" onClick={onClose}>
+          Cancel
+        </Button>
+      </Stack>
```

Comment:
Could we switch the order of the buttons and use `row` instead of `row-reverse`. I guess it's the same thing, it just feels a bit backwards to me

```suggestion
      <Stack direction="row" gap={2} sx={{ marginTop: 4 }}>
        <Button buttonType="secondary" onClick={onClose}>
          Cancel
        </Button>
        <Button
          buttonType="primary"
          disabled={!selectedConfigId}
          onClick={() => upgradeToLinodeInterfaces(state.isDryRun)}
        >
          {state.isDryRun ? 'Upgrade Dry Run' : 'Upgrade Interfaces'}
        </Button>
      </Stack>
```

Reply:
will do! switching to box/flex - row started moved all the buttons to the left side